src/store.js

```diff
@@ -177,7 +177,20 @@ export default new Vuex.Store({
 		},
 		// uiForPath returns the UI overrides for the given path (if any)
 		uiForPath: (state) => (path) => {
-			return state.hints[path.replace(/(\/|^)[0-9]+(\/|$)/g, "$1*$2")] || {}
+			const searchPath = path
+				.split('/')
+				.filter(key => key !== '')
+				.map((key, index, array) => {
+					if (isNaN(key)) {
+						return key
+					} else if (array[index - 1] === 'oneOf') {
```

Comment:
what if index is 0?

Reply:
First key can never be a number so this should not be a problem. This is because there cannot be "indexed" content without something containing the "indexed" content. If this happens it should throw an index out of bounds error.